### PR TITLE
Added TESSDATA to README.md

### PR DESCRIPTION
**Pull request by mikylucky**
Added the TESSDATA parameter description to README.md

As per the example at https://github.com/tesseract-ocr/tesstrain/wiki/GT4HistOCR#finetuning-based-on-scriptfraktur

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +93,7 @@ Run `make help` to see all the possible targets and variables:
     LEPTONICA_VERSION  Leptonica version. Default: 1.78.0
     TESSERACT_VERSION  Tesseract commit. Default: 4.1.0
     TESSDATA_REPO      Tesseract model repo to use. Default: _best
+    TESSDATA           Name of the traineddata file to start finetuning from (without file extension)
```
Comment (stweil) [inline README.md]: The name of the traineddata file is `STARTMODEL.traineddata`.

`TESSDATA` is the name of the directory of the traineddata file.

Comment (stweil) [reply README.md]: If we suggest `TESSDATA` as a variable which may be set by the user, that should also be documented in the `Makefile`.

Comment (mikylucky) [reply README.md]: Done! Let me know if it's ok

File: Makefile [hunk 2]
```diff
@@ -79,6 +79,7 @@ help:
 	@echo "    LEPTONICA_VERSION  Leptonica version. Default: $(LEPTONICA_VERSION)"
 	@echo "    TESSERACT_VERSION  Tesseract commit. Default: $(TESSERACT_VERSION)"
 	@echo "    TESSDATA_REPO      Tesseract model repo to use. Default: $(TESSDATA_REPO)"
+	@echo "    TESSDATA      	  Path to the .traineddata directory to start finetuning from. Default: $(LOCAL)/share/tessdata
```
Comment (stweil) [inline Makefile]: Please use space characters here instead of TAB.

Line 9 also still needs a comment now.

File: README.md [hunk 3]
```diff
@@ -93,7 +93,7 @@ Run `make help` to see all the possible targets and variables:
     LEPTONICA_VERSION  Leptonica version. Default: 1.78.0
     TESSERACT_VERSION  Tesseract commit. Default: 4.1.0
     TESSDATA_REPO      Tesseract model repo to use. Default: _best
-    TESSDATA           Name of the traineddata file to start finetuning from (without file extension)
+    TESSDATA           Path to the .traineddata directory to start finetuning from. Default: './usr/share/tessdata'
```
Comment (stweil) [inline README.md]: As the other descriptions don't use apostrophes, I suggest to omit them here, too.

**Comment by mikylucky** [comment]
Thank you for you support!

**Pull request merged**